recipe/meta.yaml

```diff
@@ -1,17 +1,18 @@
+{% set name = "ruamel.yaml" %}
```

Comment:
There is already a ruamel.yaml package

Reply:
This doesn't set the name of the package.
It just sets a variable that's used in the source/url templates.

Reply:
Yes, but you use the same source as another package that is already on conda-forge.  If that is really the case, this whole recipe should just be an alias package instead

Reply:
This is just a fix for #64, so that this package can start updating again. There's a whole different discussion about whether this package should exist or not:

* #1 
* https://github.com/conda-forge/ruamel.yaml-feedstock/issues/7
* https://github.com/conda-forge/ruamel_yaml-feedstock/pull/65#issuecomment-585729353
* https://github.com/conda-forge/ruamel_yaml-feedstock/pull/65#discussion_r382147311

Reply:
I think those are very coupled conversations

Reply:
How so?

Reply:
Is this builds exactly the same source as another feedstock, then why should both of them exist?

Reply:
Because this version has patches (I think) so it's not exactly the same (I think)

Or is there an option to remove it?

Reply:
But you are getting the source from the other package here I thought, which is why I was confused

Reply:
Given #64, would it be better to just use the sourceforge repo as the new source?